beast/physicsmodel/priormodel_functions.py

```diff
@@ -0,0 +1,177 @@
+import numpy as np
+
+__all__ = [
+    "_lognorm",
+    "_two_lognorm",
+    "_exponential",
+    "_imf_salpeter",
+    "_imf_kroupa",
+    "_imf_flat",
+]
+
+
+def _lognorm(x, max_pos, sigma=0.5, N=1.0):
+    """
+    Lognormal distribution
+
+    Parameters
+    ----------
+    xs : vector
+       x values
+
+    max_pos : float
+       Position of the lognormal function's maximum
+
+    sigma : float
+       Sigma of the lognormal function
+
+    N : floats
+       Multiplicative factor
+
+    Returns
+    -------
+    lognormal computed on the x grid
+    """
+    sqrt_2pi = 1.0 / np.sqrt(2 * np.pi)
+    mu = np.log(max_pos) + sigma ** 2
+
+    # avoid zero or negative due to log
+    (indxs,) = np.where(x > 0)
```

Comment:
In this purpose, we can just do indxs = x > 0. Do we need indices where the array has True value?

Reply:
Good point.  Updated to the a mask instead if index list.